scripts/taskmaster/optuna_hp_search/collect_trials.py

```diff
@@ -0,0 +1,111 @@
+import optuna
+import argparse
+import os
+import json
+import pandas
+import numpy
+
+RESULT_DIR = "/scratch/hl3236/jiant_results"
```

Comment:
If we'll have to reuse this at all, could you add a short header comment explaining what this is for?